README.md

```diff
@@ -29,6 +29,17 @@ Patreon: https://patreon.com/omarroth
 BTC: 356DpZyMXu6rYd55Yqzjs29n79kGKWcYrY  
 BCH: qq4ptclkzej5eza6a50et5ggc58hxsq5aylqut2npk
 
+## Why use Invidious?
+
+You cannot use YouTube without running non-free Javascript or being tracked by Google.
+Invidious is free software with which you can watch YouTube with zero requests to Google and play videos without Javascript.
```

Comment:
Looks to be not "zero" requests. Using uBlock Origin, if `googlevideo.com` domain is blocked, invidious cannot stream the video.